tests/unit/type/stream-cgroups.tcl

```diff
@@ -520,7 +520,7 @@ start_server {
 
         # In the past, this time would have been 1000+200, in order to avoid
         # timing issues, we increase the range a bit.
```

Comment:
it was hard for me to recover the state in order to understand the fix.
maybe this comment in better. in which case mirror it to the other 2 places.
```suggestion
        # before the fix in #13004, this time would have been 1200+ (i.e. more than 1200ms),
        # now it should be 1000, but in order to avoid timing issues, we increase the range a bit.
```

Reply:
thanks for the wording, it is better now